tests/unit/configuration_subsystem/data.py

```diff
@@ -244,6 +245,7 @@ def cli_data():
     ("plugin_type", "become", "become"),
     ("pull_policy", "never", "never"),
     ("set_environment_variable", "T1=A,T2=B,T3=C", {"T1": "A", "T2": "B", "T3": "C"}),
+    ("workdir", "/tmp/", "/tmp/"),
```

Comment:
@cidrblock That default looks pretty bad to me, as I know for sure that MacOS and few other systems do not allow users to write to /tmp for security reasons.